text/0000-manually-drop.md

```diff
@@ -127,18 +127,18 @@ impl Drop for FruitBox {
 }
 ```
 
-It is proposed that such code would become idiomatic for structures where fields must be dropped in
-a particular order.
+It is proposed that such pattern would become idiomatic for structures where fields must be dropped
```

Comment:
Perhaps: "this pattern" instead of "such pattern"? Alternates would be "such patterns" and "such a pattern", but "this" seems more natural here.